web_ui/src/pages/landing-page/workspaces-tabs/workspaces-tabs.component.tsx

```diff
@@ -7,129 +7,93 @@ import { isOrganizationAdmin } from '@geti/core/src/users/user-role-utils';
 import { RESOURCE_TYPE } from '@geti/core/src/users/users.interface';
 import { useWorkspacesApi } from '@geti/core/src/workspaces/hooks/use-workspaces.hook';
 import { WorkspaceEntity } from '@geti/core/src/workspaces/services/workspaces.interface';
-import { ActionButton, Flex, Item, Loading, TabList, TabPanels, Tabs, Tooltip, TooltipTrigger } from '@geti/ui';
-import { Add } from '@geti/ui/icons';
 import { useOverlayTriggerState } from 'react-stately';
 
 import { useOrganizationIdentifier } from '../../../hooks/use-organization-identifier/use-organization-identifier.hook';
 import { CustomTabItem } from '../../../shared/components/custom-tab-item/custom-tab-item.component';
 import { HasPermission } from '../../../shared/components/has-permission/has-permission.component';
 import { OPERATION } from '../../../shared/components/has-permission/has-permission.interface';
-import { TabItem } from '../../../shared/components/tabs/tabs.interface';
-import { hasEqualId } from '../../../shared/utils';
+import { ManagedTabs } from '../../../shared/components/managed-tabs/managed-tabs.component';
 import { CreateWorkspaceDialog } from '../../user-management/workspaces/create-workspace-dialog/create-workspace-dialog.component';
 import { LandingPageWorkspace as Workspace } from '../landing-page-workspace/landing-page-workspace.component';
 import { NoPermissionPlaceholder } from './components/no-permission-placeholder.component';
 import { CustomTabItemWithMenu } from './custom-tab-item-with-menu.component';
 import { useWorkspacesTabs } from './hooks/use-pinned-collapsed-workspace.hook';
 
-import classes from '../../../shared/components/custom-tab-item/custom-tab-item.module.scss';
+const WorkspaceItem = ({ workspace }: { workspace: WorkspaceEntity }) => {
```

Comment:
Minor: I'd rename this to `WorkspaceTabItem`, just to emphasize it will render a tab item.